app/controllers/admin/api/api_docs_backend_apis_controller.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+class Admin::Api::ApiDocsBackendApisController < Admin::Api::BaseController
```

Comment:
Maybe put this with the other `BackendApis`-namespaced controllers, inheriting from `app/controllers/admin/api/backend_apis/base_controller.rb`

Reply:
the thing is that it authorize a different action, according the existing: `app/controllers/admin/api/api_docs_services_controller.rb` and it also does not load the backend API object so it would inherit from that controller but wouldn't use it for anything.